pkg/crc/api/client_test.go

```diff
@@ -8,37 +8,44 @@ import (
 )
 
 type mockClient struct {
+	failing bool
 }
 
-func (c mockClient) Delete(deleteConfig machine.DeleteConfig) (machine.DeleteResult, error) {
+func (c *mockClient) Delete(deleteConfig machine.DeleteConfig) (machine.DeleteResult, error) {
 	return machine.DeleteResult{}, errors.New("not implemented")
 }
 
-func (c mockClient) GetConsoleURL(consoleConfig machine.ConsoleConfig) (machine.ConsoleResult, error) {
+func (c *mockClient) GetConsoleURL(consoleConfig machine.ConsoleConfig) (machine.ConsoleResult, error) {
 	return machine.ConsoleResult{}, errors.New("not implemented")
 }
 
-func (c mockClient) GetProxyConfig(machineName string) (*network.ProxyConfig, error) {
+func (c *mockClient) GetProxyConfig(machineName string) (*network.ProxyConfig, error) {
 	return nil, errors.New("not implemented")
 }
 
-func (c mockClient) IP(ipConfig machine.IPConfig) (machine.IPResult, error) {
+func (c *mockClient) IP(ipConfig machine.IPConfig) (machine.IPResult, error) {
 	return machine.IPResult{}, errors.New("not implemented")
 }
 
-func (c mockClient) PowerOff(powerOff machine.PowerOffConfig) (machine.PowerOffResult, error) {
+func (c *mockClient) PowerOff(powerOff machine.PowerOffConfig) (machine.PowerOffResult, error) {
 	return machine.PowerOffResult{}, errors.New("not implemented")
 }
 
-func (c mockClient) Start(startConfig machine.StartConfig) (machine.StartResult, error) {
+func (c *mockClient) Start(startConfig machine.StartConfig) (machine.StartResult, error) {
 	return machine.StartResult{}, errors.New("not implemented")
 }
 
-func (c mockClient) Stop(stopConfig machine.StopConfig) (machine.StopResult, error) {
+func (c *mockClient) Stop(stopConfig machine.StopConfig) (machine.StopResult, error) {
```

Comment:
This could already be a `c *mockClient` when introduced in the previous commit I believe?

Reply:
done :+1: